components/ArticleItem.js

```diff
@@ -6,12 +6,14 @@ import { listItemStyle } from './ListItem.styles';
 export default function ArticleItem({ article }) {
   return (
     <Link route="article" params={{ id: article.get('id') }}>
-      <a className="item">
-        <div className="item-text">{article.get('text')}</div>
-        <ArticleInfo article={article} />
+      <li className="item">
+        <a>
```

Comment:
`<a>` tag must be the direct children of `<Link>` so that `<Link>` can properly configure `href` property.

Please refer to the image below:

## master branch  - `<Link>` is the direct parent of `<a>`

Note that `<a>` contains `href` here, so that search engines can pick up the links easily.
![2017-11-08 1 13 46](https://user-images.githubusercontent.com/108608/32507477-21212f5c-c3ad-11e7-8b99-52c6f8059e71.png)

## This PR - No href in `<a>` tag

Although the links are still clickable, it was [handled by onClick event within next.js](https://github.com/zeit/next.js/#using-a-component-that-support-onclick). Search engines may not be able to pick the links up properly.

![2017-11-08 1 14 29](https://user-images.githubusercontent.com/108608/32507521-46f2cfb0-c3ad-11e7-8ab1-782513b65a5d.png)


In order to fix this, we should use the following structure, so that
* `<li>` is `<ul>`'s direct child
* `<a>` is `<Link>`'s direct child
* Only 1 child is allowed in `<Link>`

```js
    <li className="item">
      <Link route="article" params={{ id: article.get('id') }}>
        <a>
          <div className="item-text">{article.get('text')}</div>
          <ArticleInfo article={article} />
        </a>
      </Link>
      <style jsx>{listItemStyle}</style>
    </li>
```

Note that in this way, we should also `listItemStyle` must be edited to remove the underline of hyperlinks.

Reply:
make sense, Great!